src/components/problemCard/ProblemCard.css

```diff
@@ -0,0 +1,80 @@
+.codeutsava__section8-problems-container1 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 4rem;
+  width: 100%;
+}
```

Comment:
Use `display : grid` instead of flex, with `grid-template-columns: repeat(2 ,1fr)` for keeping 2 items per row.

Reply:
done